### DISCUSSION TITLE
Since Sphinx 5.2.0 --color-toc-background is inconsistent for autodoc entries in the toc

### DISCUSSION THREAD

**Discussion by ReimundR** [pradyunsg/furo#574] [Potential Issue] [answered]
Hi!
Thanks for the great Sphinx theme!!
Since Sphinx 5.2.0 toc entries generated by autodoc do not change to the toc background, but still have the default background color.
Is it a new bug or is there something to change?
Or anyone knows how to disable the generation of the autodoc entries for the toc? I could not find a solution.

Here's the custom CSS:
```
body {
  --color-sidebar-background: #caf5ff;
  --color-toc-background: var(--color-sidebar-background);
}
```
Config is:
```
extensions = [
    'sphinx.ext.autodoc',
    'sphinx.ext.autosummary',
    'autodocsumm',
    'sphinx_paramlinks',
    'sphinx_tabs.tabs',
    'sphinx.ext.napoleon'
]
autodoc_member_order = 'groupwise'
autoclass_content = "class"
pygments_style = 'sphinx'
html_theme = 'furo'
```

**Comment by pradyunsg** [comment, marked as answer]
Those are inline code and have background as if they’re inline code.
